lib/awesome_spawn/spec_helper.rb

```diff
@@ -30,15 +30,13 @@ def stub_bad_run!(command, options = {})
     private
 
     def stub_run(mode, method, command, options)
-      output = options[:output] || ""
-      error  = options[:error]  || (mode == :bad ? "Failure" : "")
-      exit_status = options[:exit_status] || (mode == :bad ? 1 : 0)
+      output = options.delete(:output) || ""
```

Comment:
You probably want to `output = output.dup` because `#delete` will modify the original.